### ISSUE TITLE
False positives that are already fixed upstream

### ISSUE THREAD

**Issue by Olexorus** [ChenCMD/datapack-linter#12]
I just found this, it's really cool, thanks for making this!

However it seems like it's not up to date with the VS Code plugin, since some syntax is incorrectly detected as wrong even though it's working in the VS Code.  An example of this is
```
gamerule playersSleepingPercentage 33
```

You can see the failed run here: https://github.com/Olexorus/PandamiumDatapack/actions/runs/1201405430
(Some of those issues are also wrongly detected in VS Code)

**Comment by ChenCMD** [comment]
We have just released v1.3.4. Can you please check if the problem still occurs using it?

**Comment by Olexorus** [comment]
Updating to 1.3.4 fixed it, thank you!
